src/Runtime/Runtime/System.Windows.Controls/PasswordBoxView.cs

```diff
@@ -35,6 +36,7 @@ internal PasswordBoxView(PasswordBox host)
             }
 
             Host = host;
+            Host.Unloaded += (s, e) => DisposeJsCallbacks();
```

Comment:
It is better to listen to the PasswordBoxView Unloaded event. In the case that the host changes its template, its unloaded event will not be raised (but the one of the PasswordBoxView will because a new view is generated for each template (cf. PasswordBox.OnApplyTemplate(...), and the previous one is discarded)
```suggestion
            Unloaded += (s, e) => DisposeJsCallbacks();
```

Reply:
Thanks for your suggestion, I updated the code